package.json

```diff
@@ -74,8 +74,9 @@
     "pretest": "npm run build",
     "posttest": "agadoo internal/index.mjs",
     "prepublishOnly": "npm run lint && PUBLISH=true npm test",
-    "tsd": "tsc -p src/compiler --emitDeclarationOnly && tsc -p src/runtime --emitDeclarationOnly",
-    "lint": "eslint \"{src,test}/**/*.{ts,js}\""
+    "tsd": "tsc -p src/compiler --emitDeclarationOnly --strict false && tsc -p src/runtime --emitDeclarationOnly --strict false",
+    "tsc-strict-error": "node ci-script/tsc-strict-error.mjs --unhandled-rejections=strict",
```

Comment:
does this `unhandled-rejections` do anything? I don't see it present in the script?

Reply:
https://nodejs.org/api/process.html#process_event_uncaughtexception It's my extra caution when working with async/await/Promise. It's also easier to debug. It has helped me figured out some weird things during the work of this PR, such as cannot read files.